server/planning/output_formatters/json_planning.py

```diff
@@ -178,12 +179,11 @@ def _expand_delivery(self, coverage):
             if delivery.get("item_state") == CONTENT_STATE.PUBLISHED:
                 item_never_published = False
 
-        if item_never_published:
-            deliveries = []
-
         return deliveries, assignment.get("assigned_to").get("state")
 
     def _expand_coverage_contacts(self, coverage):
+        EXTENDED_INFO = bool(app.config.get("PLANNING_JSON_ASSIGNED_INFO_EXTENDED"))
```

Comment:
It would be good to add this to the README.md file (maybe under a new `Publishing` heading: https://github.com/superdesk/superdesk-planning?tab=readme-ov-file#config).
That way Planning configs are at least documented somewhere